code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -615,23 +629,19 @@ Then we space some of our heat, and think about if we should stop conducting.
 	return ..()
 
 /turf/proc/radiate_to_spess() //Radiate excess tile heat to space
-	if(temperature > T0C) //Considering 0 degC as te break even point for radiation in and out
-		var/delta_temperature = (temperature_archived - TCMB) //hardcoded space temperature
-		if((heat_capacity > 0) && (abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER))
-
-			var/heat = thermal_conductivity*delta_temperature* \
-				(heat_capacity*HEAT_CAPACITY_VACUUM/(heat_capacity+HEAT_CAPACITY_VACUUM))
-			temperature -= heat/heat_capacity
+	if(temperature <= T0C) //Considering 0 degC as te break even point for radiation in and out
+		return
+	var/delta_temperature = (temperature_archived - TCMB) //hardcoded space temperature
+	if(heat_capacity > 0 && abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER)
+		var/heat = thermal_conductivity * delta_temperature * (heat_capacity * HEAT_CAPACITY_VACUUM / (heat_capacity + HEAT_CAPACITY_VACUUM))
+		temperature -= heat/heat_capacity
 
 /turf/open/proc/temperature_share_open_to_solid(turf/sharer)
 	sharer.temperature = air.temperature_share(null, sharer.thermal_conductivity, sharer.temperature, sharer.heat_capacity)
 
 /turf/proc/share_temperature_mutual_solid(turf/sharer, conduction_coefficient) //This is all just heat sharing, don't get freaked out
 	var/delta_temperature = (temperature_archived - sharer.temperature_archived)
 	if(abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER && heat_capacity && sharer.heat_capacity)
-
-		var/heat = conduction_coefficient*delta_temperature* \
```

Comment:
Don't like the \s? I'm alright with them myself, think they make the lines less hellish to read

Reply:
See https://github.com/tgstation/tgstation/pull/57112